docs/DeveloperGuide.md

```diff
@@ -418,7 +479,7 @@ and the **Actor** is the `user`, unless specified otherwise)
     
         Use case resumes at step 2.
 
-* 4a. The list is empty.
+* 3a. Either one or both lists are empty.
```

Comment:
I am not too sure about this, whether the lists are empty or not, the use cases ends there right? So is there a point in having this extension?

Reply:
Honestly I'm also not sure but the contacts one has it